tests/baselines/reference/tsxAttributeResolution3.errors.txt

```diff
@@ -44,7 +41,7 @@ tests/cases/conformance/jsx/file.tsx(31,8): error TS2322: Type 'number' is not a
     // Error
     var obj5 = { x: 32, y: 32 };
     <test1 x="ok" {...obj5} />

```

Comment:
this is basically the JSX-equivalent of the repro case from the linked issue - that's why I didn't add any extra test case for JSX